tests/cypress/integration/notifications.cy.js

```diff
@@ -0,0 +1,60 @@
+// <reference types="Cypress" />
+const marketplace = require('../fixtures/notifications.json');
+
+describe('Notifications', function () {
+
+	before(() => {
+		cy.server();
```

Comment:
@circlecube We can remove this line. See https://docs.cypress.io/api/commands/intercept#Comparison-to-cy-route:~:text=does%20not%20require%20calling%20cy.server()%20before%20use%20%2D%20in%20fact%2C%20cy.server()%20does%20not%20influence%20cy.intercept()%20at%20all.